### PR TITLE
[DOC-562] Update build the docs page with instructions for Ubuntu and a specific hugo version.

### PR DESCRIPTION
**Pull request by sudhanshu456**
@netlify /preview/contribute/docs/docs-build/

### CODE REVIEW

File: docs/content/preview/contribute/docs/macos.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+    Xcode is many gigabytes. Install the command-line tools unless you actually need the full Xcode.
+
+* [**Homebrew**](https://brew.sh) on macOS.
+
+* **Python** (version 3.10 or earlier): You need `python` to be available somewhere in your shell path.
+
+    Recent versions of macOS have only a `python3` executable, as does the Homebrew install. You can use [pyenv](https://github.com/pyenv/pyenv) to manage multiple versions of python on your system. Make sure to point to Python version 3.10 or earlier.
+
+* **Hugo**: `brew install hugo` installs the latest version.
```
Comment (aishwarya24) [inline docs/content/preview/contribute/docs/macos.md:21]: So for MacOS, add these instructions to install a specific version. Lets fix 0.125.6 as thats the current one we are using :
Include the following steps to install a specific hugo version. (0.125.6)
• Unpin hugo - brew unpin hugo
• Uninstall older version - brew uninstall hugo
• Download the attached file.
• brew install hugo.rb
• Prevent automatic updates with brew pin hugo

Comment (sudhanshu456) [reply docs/content/preview/contribute/docs/macos.md]: Added the steps for v0.131.0 version of Hugo. Please check.

File: docs/content/preview/contribute/docs/macos.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    Xcode is many gigabytes. Install the command-line tools unless you actually need the full Xcode.
+
+* [**Homebrew**](https://brew.sh) on macOS.
+
+* **Python** (version 3.10 or earlier): You need `python` to be available somewhere in your shell path.
+
+    Recent versions of macOS have only a `python3` executable, as does the Homebrew install. You can use [pyenv](https://github.com/pyenv/pyenv) to manage multiple versions of python on your system. Make sure to point to Python version 3.10 or earlier.
+
+* **Hugo**: Install Hugo `v0.131.0` version. Follow these steps:
```
Comment (ddhodge) [inline docs/content/preview/contribute/docs/macos.md:21]: stick with the current one v0.125.6

File: docs/content/preview/contribute/docs/macos.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+  * Unpin Hugo to stop its formula from being updated - `brew unpin hugo`
+  * Uninstall any older version if installed - `brew uninstall hugo`
+  * Download the `v0.131.0` formula file from [Homebrew's repository](https://github.com/Homebrew/homebrew-core/blob/69dfbc7747a766d9e7e2a94d57adacae359d8b5a/Formula/h/hugo.rb).
```
Comment (ddhodge) [inline docs/content/preview/contribute/docs/macos.md:25]: v0.125.6

File: docs/content/preview/contribute/docs/ubuntu.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<!--
++++
+private=true
++++
+-->
+
+* **Python** (version 3.10 or earlier): You need `python` to be available somewhere in your shell path.
+
+    Recent versions of Ubuntu default to `python` and point to Python 3. If not, you can install new version of Python, and use [pyenv](https://github.com/pyenv/pyenv) to manage multiple Python versions. Ensure you're using Python 3.10 or earlier.
```
Comment (ddhodge) [inline docs/content/preview/contribute/docs/ubuntu.md:9]:
```suggestion
    Recent versions of Ubuntu default to `python` and point to Python 3. If not, you can install a new version of Python, and use [pyenv](https://github.com/pyenv/pyenv) to manage multiple Python versions. Ensure you're using Python 3.10 or earlier.
```

File: docs/content/preview/contribute/docs/ubuntu.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+* **Hugo**: Install `hugo_extended_0.131.0` version from the [official Hugo releases](https://github.com/gohugoio/hugo/releases) as per your Linux machine architecture e.g `amd`, `arm`, and `64bit`. Make sure the `hugo` binary is available in the shell path.
```
Comment (ddhodge) [inline docs/content/preview/contribute/docs/ubuntu.md:11]:
```suggestion
* **Hugo**: Install the `hugo_extended_0.125.6` version from the [official Hugo releases](https://github.com/gohugoio/hugo/releases) for your Linux machine architecture (for example, `amd`, `arm`, `64bit`). Make sure the `hugo` binary is available in the shell path.
```

File: docs/content/preview/contribute/docs/ubuntu.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+* **Go**: Install latest Go from the [official Go website](https://golang.org/dl/).
+
+* **Git client**: Ubuntu includes Git by default, but to ensure you have the latest version (`sudo apt-get install git`).
```
Comment (ddhodge) [inline docs/content/preview/contribute/docs/ubuntu.md:15]:
```suggestion
* **Git client**: Ubuntu includes Git by default, but ensure you have the latest version (`sudo apt-get install git`).
```

**Review by ddhodge — APPROVED** [review]
Minor comments

File: docs/content/preview/contribute/docs/macos.md (hunk 2 above)
Comment (sudhanshu456) [reply docs/content/preview/contribute/docs/macos.md:21]:
``` suggestion
* **Hugo**: Install Hugo `v0.125.6` version. Follow these steps:
```

File: docs/content/preview/contribute/docs/macos.md (hunk 3 above)
Comment (sudhanshu456) [reply docs/content/preview/contribute/docs/macos.md:25]:
```suggestion
  * Download the `v0.125.6` formula file from [Homebrew's repository](https://github.com/Homebrew/homebrew-core/blob/d6cdfadac69489a73e65b1ef46f59382836219c4/Formula/h/hugo.rb).
```

File: docs/content/preview/contribute/docs/macos.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+    ```sh
+    $ xcode-select --install
+    ```
+
+    Xcode is many gigabytes. Install the command-line tools unless you actually need the full Xcode.
+
+* [**Homebrew**](https://brew.sh) on macOS.
+
+* **Python** (version 3.10 or earlier): You need `python` to be available somewhere in your shell path.
```
Comment (ddhodge) [inline docs/content/preview/contribute/docs/macos.md:17]:
```suggestion
* **Python** (v3.10 or earlier): You need `python` to be available somewhere in your shell path.
```

File: docs/content/preview/contribute/docs/macos.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+
+    Recent versions of macOS have only a `python3` executable, as does the Homebrew install. You can use [pyenv](https://github.com/pyenv/pyenv) to manage multiple versions of python on your system. Make sure to point to Python version 3.10 or earlier.
```
Comment (ddhodge) [inline docs/content/preview/contribute/docs/macos.md:19]:
```suggestion
    Recent versions of macOS have only a `python3` executable, as does the Homebrew install. You can use [pyenv](https://github.com/pyenv/pyenv) to manage multiple versions of python on your system. Make sure to point to Python v3.10 or earlier.
```

File: docs/content/preview/contribute/docs/macos.md [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+
+* **Hugo**: Install Hugo `v0.125.6` version. Follow these steps:
```
Comment (ddhodge) [inline docs/content/preview/contribute/docs/macos.md:21]:
```suggestion
* **Hugo**: Install Hugo v0.125.6. Follow these steps:
```

File: docs/content/preview/contribute/docs/macos.md [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+
+  * Unpin Hugo to stop its formula from being updated - `brew unpin hugo`
+  * Uninstall any older version if installed - `brew uninstall hugo`
+  * Download the `v0.125.6` formula file from [Homebrew's repository](https://github.com/Homebrew/homebrew-core/blob/d6cdfadac69489a73e65b1ef46f59382836219c4/Formula/h/hugo.rb).
```
Comment (ddhodge) [inline docs/content/preview/contribute/docs/macos.md:25]:
```suggestion
  * Download the v0.125.6 formula file from [Homebrew's repository](https://github.com/Homebrew/homebrew-core/blob/d6cdfadac69489a73e65b1ef46f59382836219c4/Formula/h/hugo.rb).
```

File: docs/content/preview/contribute/docs/ubuntu.md [hunk 11]
```diff
@@ -0,0 +1,15 @@
+<!--
++++
+private=true
++++
+-->
+
+* **Python** (version 3.10 or earlier): You need `python` to be available somewhere in your shell path.
```
Comment (ddhodge) [inline docs/content/preview/contribute/docs/ubuntu.md:7]:
```suggestion
* **Python** (v3.10 or earlier): You need `python` to be available somewhere in your shell path.
```

**Review by ddhodge — APPROVED** [review]

**Pull request merged**